workflow-trigger

```diff
@@ -1 +1,2 @@
-2023-07-20T15:30:00Z
\ No newline at end of file
+2023-07-20T15:30:00Z
+2023-10-04T15:30:00Z
```

Comment:
Hi @BenjaminSsempala , thanks for this.

I am not sure about the purpose of this PR. 
I sense that this file is usually changes when we want to redeploy ALL the apps in the frontend repository.
Question: do we want to trigger deployment of Netmanager ONLY or the entire frontend repo, that is (website, Analytics, Calibrate, etc)?

In case we want to trigger only one thing, I suggest that we just make random small changes to the README file of that repo accordingly.

Thanks again!